modules/installation-azure-permissions.adoc

```diff
@@ -6,10 +6,11 @@
 [id="installation-azure-permissions_{context}"]
 = Required Azure roles
 
-Your Microsoft Azure account must have the following roles for the subscription
+You can use your Microsoft Azure account for `az login` and browsing in the web console. By using your Microsoft Azure account you can run the installer and create the cluster. Your Microsoft Azure account must have the following roles for the subscription
```

Comment:
> By using your Microsoft Azure account you can run the installer and create the cluster

I think the statement is wrong, The Microsoft Azure account has the correct permissions to create the service principal, the created service principal account will be used to run the installer and create the cluster.

Reply:
Changed the sentence. Managed to maintain the present tense. Let me know if any change is required!